Content.Client/Options/UI/OptionsMenu.xaml.cs

```diff
@@ -20,7 +20,7 @@ public OptionsMenu()
             Tabs.SetTabTitle(2, Loc.GetString("ui-options-tab-controls"));
             Tabs.SetTabTitle(3, Loc.GetString("ui-options-tab-audio"));
             Tabs.SetTabTitle(4, Loc.GetString("ui-options-tab-network"));
-            Tabs.SetTabTitle(5, Loc.GetString("ui-options-tab-deltav")); // DeltaV specific settings
+            Tabs.SetTabTitle(5, Loc.GetString("ui-options-tab-einsteinengines")); // DeltaV specific settings
```

Comment:
```suggestion
            Tabs.SetTabTitle(5, Loc.GetString("ui-options-tab-einsteinengines"));
```

Reply:
Yea we could remove the tab outright if the species vision is not applied.